Lib/opcode.py

```diff
@@ -211,5 +211,6 @@ def jabs_op(name, op):
 def_op('SET_UPDATE', 163)
 def_op('DICT_MERGE', 164)
 def_op('DICT_UPDATE', 165)
+def_op('INT_ADD', 166)
 
```

Comment:
Consider `ADD_INT` as the instruction adds an int to its operand, and does not imply that its operand is an int.